### PR TITLE
feat(tokenfactory): execute msgs for CreateDenom, ChangeAdmin, UpdateModuleParams

### PR DESCRIPTION
**Pull request by Unique-Divine**
- Includes queries, transaction messages, and CLI commands for CreateDenom, ChangeAdmin, and UpdateModuleParams
- Part of #1584 
- Includes fix from #1612 (preferably, merge this one first to avoid conflicts)

### CODE REVIEW

**Comment by Unique-Divine** [comment]
I'm encountering something strange here where it says the perp protos are breaking and I'm not even editing those files.

image

image

File: x/tokenfactory/keeper/grpc_query.go [hunk 1]
```diff
@@ -38,3 +39,47 @@ func (q Querier) Params(
 	}
 	return &types.QueryParamsResponse{Params: params}, nil
 }
+
+// Denoms: Returns all registered denoms for a given creator.
+func (q Querier) Denoms(
```
Comment (jgimeno) [inline x/tokenfactory/keeper/grpc_query.go:44]: I dont see why we need this and the other one, they are very similar

File: x/tokenfactory/keeper/msg_server.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	grpcstatus "google.golang.org/grpc/status"
+
+	"github.com/NibiruChain/nibiru/x/tokenfactory/types"
+)
+
+var _ types.MsgServer = (*Keeper)(nil)
+
+var errNilMsg error = grpcstatus.Errorf(grpccodes.InvalidArgument, "nil msg")
+
+func (k Keeper) CreateDenom(
```
Comment (jgimeno) [inline x/tokenfactory/keeper/msg_server.go:18]: I think we created this without thinking the use case and how it aligns with our chain.

File: x/tokenfactory/keeper/grpc_query.go (hunk 1 above)
Comment (Unique-Divine) [reply x/tokenfactory/keeper/grpc_query.go:44]: It's the DenomsFromCreator query

**Review by jgimeno — APPROVED** [review]

**Pull request merged**